### ISSUE TITLE
Requests a weird indentation when we have references and class

### ISSUE THREAD

**Issue by pvalsecc** [adrienverge/yamllint#78]
This YAML:

```yaml
scale: !toto &tutu
  default: "Massstab"
```

Leads to a weird error message:

```
./test.yaml
  2:3       error    wrong indentation: expected 7 but found 2  (indentation)
```

**Comment by adrienverge** [comment]
I agree there is a bug here. Both
```yaml
scale: &tutu
  default: "Massstab"
```
and
```yaml
scale: !toto
  default: "Massstab"
```
are reported correct, but the combination `!toto &tutu` leads to an error that should not exist.

Contributions to solve this are welcome!

**Comment by pvalsecc** [comment]
I've tried, but the problem lies here:
https://github.com/adrienverge/yamllint/blob/master/yamllint/rules/indentation.py#L445

That if should be a while, but that would require us to have nextnextnext or even nextnextnext. I really don't know enough of the code to know how to fix that.

I event have a UT for it, if you want: https://github.com/pvalsecc/yamllint/commit/8a742d2f2b86bac8781eefb0087beec4da1a2239

By the way, I have some other UTs failing with python 3.5.2:
```
FAIL: test_run_module_on_dir (tests.test_module.ModuleTestCase)
FAIL: test_run_module_on_bad_dir (tests.test_module.ModuleTestCase)
FAIL: test_run_module_no_args (tests.test_module.ModuleTestCase)
ERROR: test_run_module_on_file (tests.test_module.ModuleTestCase)
```
